wordpress-mobile/WordPress-iOS
# [SIWA] Existing accounts: show user info in epilogue

Fixes #12434 
WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/128

To test:
- Go to either Log In or Sign Up.
- Select Continue with Apple.
- Log in with an existing Apple account.
- Verify the epilogue now shows the display name and username.

<img width="400" alt="epilogue" src="https://user-images.githubusercontent.com/1816888/64298824-0f182380-cf34-11e9-89c2-34ea84169092.png">

Update release notes:
